rllib/evaluation/tests/test_worker_set.py

```diff
@@ -0,0 +1,78 @@
+import gym
+import unittest
+
+import ray
+from ray.rllib.algorithms.algorithm_config import AlgorithmConfig
+from ray.rllib.evaluation.worker_set import WorkerSet
+from ray.rllib.examples.policy.random_policy import RandomPolicy
+from ray.rllib.policy.sample_batch import DEFAULT_POLICY_ID
+
+
+class TestWorkerSet(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        ray.init()
+
+    @classmethod
+    def tearDownClass(cls):
+        ray.shutdown()
+
+    def test_foreach_worker(self):
+        """Test to make sure basic sychronous calls to remote workers work."""
+        ws = WorkerSet(
+            env_creator=lambda _: gym.make("CartPole-v1"),
+            default_policy_class=RandomPolicy,
+            config=AlgorithmConfig().rollouts(num_rollout_workers=2),
+            num_workers=2,
+        )
+
+        policies = ws.foreach_worker(
```

Comment:
Nit: Maybe we want to get some property of the policies instead? It's not too common to pull the entire policy object through a ray.get :), but up to you to leave this test as-is.

Reply:
ah, I would if this was not a unit test.
I guess for testing, this should be fine?